build.gradle

```diff
@@ -1,20 +1,32 @@
+
 plugins {
-    id 'io.github.gradle-nexus.publish-plugin' version '1.3.0'
+    id "java-library"
+    id "org.grails.internal.grails-plugin-publish"
+    id "org.grails.grails-gsp"
+    id "org.grails.grails-plugin"
 }
 
-ext.isSnapshot = project.projectVersion.endsWith('-SNAPSHOT')
-ext.isReleaseVersion = !isSnapshot
-ext."signing.keyId" = project.hasProperty("signing.keyId") ? project.getProperty('signing.keyId') : System.getenv('SIGNING_KEY')
-ext."signing.secretKeyRingFile" = project.hasProperty("signing.secretKeyRingFile") ? project.getProperty('signing.secretKeyRingFile') : "${System.properties['user.home']}${File.separator}.gnupg${File.separator}secring.gpg"
-ext."signing.password" = project.hasProperty("signing.password") ? project.getProperty('signing.password') : System.getenv('SIGNING_PASSPHRASE')
-
 version = project.projectVersion
+group = 'org.grails.plugins'
 
 allprojects {
 
     repositories {
         mavenCentral()
         maven { url "https://repo.grails.org/grails/core" }
+        maven {
+            url = 'https://repo.gradle.org/gradle/libs-releases'
+            description = 'Needed for Gradle Tooling API'
+        }
+    }
+
+    configurations.configureEach {

```

Comment:
Now that you have updated everything, if you remove this, will it still build?  If so, please remove, if not leave it.